mypy/checker.py

```diff
@@ -5271,10 +5271,15 @@ def format_expr_type() -> str:
             else:
                 return f"Expression has type {typ}"
 
+        def get_expr_name() -> str:
+            if hasattr(expr, "name"):
```

Comment:
Let's do `isinstance(..., (NameExpr, MemberExpr))`. We prefer isinstance checks over hasattr, since e.g. we'll get a type checker warning if we change the field or those classes.

Reply:
Makes sense, updated this.